nav2_system_tests/src/planning/planner_tester.cpp

```diff
@@ -115,19 +107,19 @@ void PlannerTester::loadMap(const std::string image_file_path, const std::string
     // User can set an environment variable to the location of the test src
     char const * path = getenv("TEST_MAP");
     if (path == NULL) {
-      throw std::runtime_error("PlannerTester::loadMap: path to map image file"
+      throw std::runtime_error("Path to map image file"
               " has not been specified in environment variable `TEST_MAP`.");
     } else {
       file_path = std::string(path);
     }
   }
 
-  RCLCPP_INFO(this->get_logger(), "PlannerTester::loadMap: file_path: %s", file_path.c_str());
+  RCLCPP_INFO(this->get_logger(), "Loading map with file_path: %s", file_path.c_str());
 
   if (!yaml_file_name.empty()) {
     // TODO(orduno): #443 parse yaml file
-    RCLCPP_WARN(this->get_logger(), "PlannerTester::loadMap: yaml file parser not implemented"
-      " yet, will use default values instead");
+    RCLCPP_WARN(this->get_logger(),
+      "Yaml file parser not implemented yet, will use default values instead");
```

Comment:
I think you should exit here instead of continuing on with different values than the user specified.

Reply:
Pushed a change on this.